server/src/api/context.go

```diff
@@ -53,7 +52,35 @@ func (s *Server) BoardParticipantContext(next http.Handler) http.Handler {
 		}
 
 		user := r.Context().Value("User").(uuid.UUID)
+		banned, err := s.sessions.ParticipantBanned(r.Context(), board, user)
+		if err != nil {
+			log.Errorw("unable to check if participant is banned", "err", err)
+			common.Throw(w, r, common.InternalServerError)
+			return
+		}
+
+		if banned {
+			common.Throw(w, r, common.GoneError(errors.New("participant is currently banned from this session")))
```

Comment:
Im not sure if GoneError is the most suitable for a banning error. Maybe it should be something like access denied.
Gone would indicate that a ressource is gone for good and not just that the access to the ressource is not granted

Reply:
But the 403 Forbidden is better than 401 Unauthorized. 
401 Unauthorized is specifically for the case that the client is unauthenticated, which is not the case here.

Reply:
403 won in the end! :)